shoop/core/models/_contacts.py

```diff
@@ -296,6 +322,11 @@ def groups(self):
         return ContactGroup.objects.filter(identifier=self.default_contact_group_identifier)
 
 
+def _split_name(full_name):
+    names = full_name.rsplit(" ", 1)
```

Comment:
We could do better with regular expressions and some additional data like list of common surnames or name articles and prepositions of course, but I'm no so sure if that's really needed.